suites/pacific/upgrades/tier-2_upgrade_test-multi-path-upgrade-to-5.2-latest.yaml

```diff
@@ -0,0 +1,100 @@
+# using the configuration given in the suite
+#
+
+# RHCS 5.x multihop upgrade test suite.
+tests:
+- test:
+    name: install ceph pre-requisites
+    module: install_prereq.py
+    abort-on-fail: True
+
+- test:
+    name: upgrade the RHCS cluster based on the path specified
+    polarion-id: "CEPH-83575288"
+    module: test_upgrade.py
+    config:
+      paths:
+        4.2:
+          config:
+            ansi_config:
+              osd_scenario: lvm
+              osd_auto_discovery: False
+              copy_admin_key: True
+              dashboard_enabled: True
+              dashboard_admin_user: admin
+              dashboard_admin_password: p@ssw0rd
+              grafana_admin_user: admin
+              grafana_admin_password: p@ssw0rd
+              cephfs_pools:
+                - name: "cephfs_data"
+                  pgs: "32"
+                - name: "cephfs_metadata"
+                  pgs: "32"
+        4.3:
+          config:
+            ansi_config:
+              osd_scenario: lvm
+              osd_auto_discovery: False
+              copy_admin_key: True
+              dashboard_enabled: True
+              dashboard_admin_user: admin
+              dashboard_admin_password: p@ssw0rd
+              grafana_admin_user: admin
+              grafana_admin_password: p@ssw0rd
+              cephfs_pools:
+                - name: "cephfs_data"
+                  pgs: "32"
+                - name: "cephfs_metadata"
+                  pgs: "32"
+          upgrade_steps:
+            step:
+              command: upgrade_all
+        5.1:
+          config:
+            verify_cephadm_containers: True
+            ansi_config:
+              osd_scenario: lvm
+              osd_auto_discovery: False
+              copy_admin_key: True
+              dashboard_enabled: True
+              dashboard_admin_user: admin
+              dashboard_admin_password: p@ssw0rd
+              grafana_admin_user: admin
+              grafana_admin_password: p@ssw0rd
+              containerized_deployment: True
+              upgrade_ceph_packages: True
+              cephfs_pools:
+                - name: "cephfs_data"
+                  pgs: "32"
+                - name: "cephfs_metadata"
+                  pgs: "32"
+          upgrade_steps:
+            step:
+              command: upgrade_all
+        5.2:
+          config:
+            command: start
+            service: upgrade
+            base_cmd_args:
+              verbose: True
+            benchmark:
+              type: rados                      # future-use
+              pool_per_client: True
+              pg_num: 128
+              duration: 10
+            verify_cluster_health: True
+            verify_cephadm_containers: True
+          upgrade_steps:
+            step:
+              command: upgrade_all
+    desc: deploy ceph setup using ceph-ansible and upgrade to the versions specified in paths
```

Comment:
It's good to mention paths in the description like `Following upgrade paths 4.2 -> 4.3 -> 5.1-> 5.2` so that in the email result user can understand it easily? otherwise test name and description looks same

Reply:
Done.